sample/src/test/java/app/cash/paparazzi/sample/SimilarDeltaTest.kt

```diff
@@ -0,0 +1,62 @@
+package app.cash.paparazzi.sample
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import app.cash.paparazzi.Paparazzi
+import org.junit.Rule
+import org.junit.Test
+
+class SimilarDeltaTest {
+  @get:Rule
+  val paparazzi = Paparazzi()
+
+  @Test
+  fun composeSimilarDeltaBoxes() {
+    paparazzi.snapshot { SimilarDeltaBoxes(shouldUseOriginalBackgroundColor = true) }
+  }
+}
+
+@Suppress("TestFunctionName")
+@Composable
+fun SimilarDeltaBoxes(
+  shouldUseOriginalBackgroundColor: Boolean
+) {
+  Column(
+    Modifier
+      .background(Color.White)
+      .fillMaxSize()
+      .wrapContentSize()
+  ) {
+    Box(
+      Modifier
+        .background(if (shouldUseOriginalBackgroundColor) Color.Yellow else Color(0xFFFFFD00))
```

Comment:
`0xFFFFFD00` triggers the similar pixel check as its green value is 2 less than yellow.

Reply:
rather than a test in sample, i'd move this to a plugin test.  I'd suggest:

* copying `verify-mode-failure` to something like `verify-similar`
* copy/paste `PaparazziPluginTest.verifyFailure` to something like `PaparazziPluginTest.verifySimilar`
  * update the fixtureRoot accordingly
  * temporarily, update the argument from `verifyPaparazziDebug` to `recordPaparazziDebug`
  * you might have to make other changes, depending on whether you want to keep the test class and method names as is, or replace with yours.
* replace VerifyTest with your test here
  * might have to add compose to build.gradle, see other projects for examples
* remove the conditional logic from this test, making it the "before" case
* run the test project and copy the image from `build/reports/paparazzi/debug/images` to `src/test/snapshots/images`.  
  * This is your "golden" file.
* update the argument back to `verifyPaparazziDebug`
* modify this test to exhibit the "after" case.
* run the test project again (it should fail) and copy the delta file in `build/paparazzi/failures` to `src/test/resources`.
* run the test project again (it should pass)